apps/io/feeding_services/wufoo.py

```diff
@@ -47,7 +47,7 @@ class WufooFeedingService(FeedingService):
         }
     ]
 
-    force_values = {'feed_parser': 'wufoo'}
+    restricted_values = {'feed_parser': ['wufoo']}
```

Comment:
it there any other usecase for restricted value? I think it could be more explicit via:
```python3
feed_parser = 'wufoo'
```
but not sure if there would be a restriction to more than 1 parsers.
this should be also defined imo for tap, ritzau, rss and twitter.

Reply:
The code supports now one value lists (in which case it hides the feed parser field) and also multi-valued lists, together with tests. I don't see any logical reason to reduce the functionality once it's already done.
From the task: "we know that a feeding service can only be used with some feed parsers". I understand from this text that there may be multiple values so I implemented it this way.

Reply:
simpler code is easier to maintain and read, that would be the reason.
what about other restricted values? I don't see that in the task.